gradle-consistent-versions-idea-plugin/src/main/java/com/palantir/gradle/versions/intellij/VersionProps.flex

```diff
@@ -33,20 +33,18 @@ import com.intellij.psi.TokenType;
 
 CRLF=\R
 WHITE_SPACE=[\ \n\t\f]
-FIRST_VALUE_CHARACTER=[^ \n\f\\] | "\\"{CRLF} | "\\".
-VALUE_CHARACTER=[^\n\f\\] | "\\"{CRLF} | "\\".
-END_OF_LINE_COMMENT=("#"|"!")[^\r\n]*
+VALUE_CHARACTER=[^ \n\f]+
```

Comment:
maybe rename this from VALUE_CHARACTER to VALUE/VERSION (since now VALUE_CHARACTER matches multiple characters) and you can update the lexer rule on line 68 to `<WAITING_VALUE> {VALUE_CHARACTER}`

Reply:
same for GROUP_PART lexter rule, I think it can become `<YYINITIAL> {GROUP_PART}` (line 49)